cmd/optimizely/testdata/default.yaml

```diff
@@ -32,6 +32,10 @@ client:
   datafileURLTemplate: "https://localhost/v1/%s.json"
   eventURL: "https://logx.localhost.com/v1"
   sdkKeyRegex: "custom-regex"
+  datafileCacheService:
```

Comment:
add one more property enableDatafileCacheService and based on that, execute the logic.